atemOSC/AppDelegate.mm

```diff
@@ -457,10 +457,25 @@ - (void)cleanUpConnection
 
 - (void)sendStatus
 {
-	mDownstreamKeyerMonitor->sendStatus();
-	mMixEffectBlockMonitor->sendStatus();
-	mTransitionParametersMonitor->sendStatus();
-	mMacroPoolMonitor->sendStatus();
+	OSCMessage *newMsg = [OSCMessage createWithAddress:@"/atem/led/green"];
+	[newMsg addFloat:isConnectedToATEM ? 1.0 : 0.0];
+	[outPort sendThisMessage:newMsg];
+	newMsg = [OSCMessage createWithAddress:@"/atem/led/red"];
+	[newMsg addFloat:isConnectedToATEM ? 0.0 : 1.0];
+	[outPort sendThisMessage:newMsg];
+
+	dispatch_after(dispatch_time(DISPATCH_TIME_NOW, 0.1 * NSEC_PER_SEC), dispatch_get_main_queue(), ^{
+		mDownstreamKeyerMonitor->sendStatus();
+	});
+	dispatch_after(dispatch_time(DISPATCH_TIME_NOW, 0.2 * NSEC_PER_SEC), dispatch_get_main_queue(), ^{
+		mTransitionParametersMonitor->sendStatus();
+	});
+	dispatch_after(dispatch_time(DISPATCH_TIME_NOW, 0.3 * NSEC_PER_SEC), dispatch_get_main_queue(), ^{
+		mMacroPoolMonitor->sendStatus();
+	});
+	dispatch_after(dispatch_time(DISPATCH_TIME_NOW, 0.4 * NSEC_PER_SEC), dispatch_get_main_queue(), ^{
```

Comment:
How's it go with a delay of 50ms instead of a delay of 100ms each? My concern is that 100ms for each adds up quite quickly. A 400ms delay is readily noticeable for instance.

Reply:
In testing, it's not too bad, noticeable but not unexpected.  As we add more feedback though, I might try 50ms, cause it could quickly get out of hand

Reply:
👍

I assume it would eventually get to a point where if you made the delay too low, you would end up having the original issue again.

I'm happy to have this number adjusted if needed as we add more notifications.

Reply:
Agreed, this feels more like a band-aid, I think I have a better idea of how to generalize it in the future, but for now working is good.  Thanks for the suggestion.